packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -284,7 +284,7 @@ function stateReducer(state, actionAndChanges) {
   }
 }
 
-const Dropdown = React.forwardRef(
+const Dropdown = React.forwardRef<HTMLButtonElement, DropdownProps<any>>(
```

Comment:
The usage of `any` here overrides `ItemType`. Paste this code at the bottom of this file:

```tsx
export const Test = () => (
  <div>
    <Dropdown
      label="placeholder"
      id="dropdown"
      items={[]}
      titleText={<div />}
    />
  </div>
);
```

Then hover over `items` and see that the intellisense tooltips says

```
(property) DropdownProps<any>.items: any[]
```

Then, if I revert all of the changes in this file and again paste the code snippet above into it, the intellisense tooltip says:

```
(property) items: unknown[]
```

So it seems like TypeScript support is regressing with these changes.

Reply:
I'll just revert the change on that line.  The important thing is the type that's exported from DropDown.tsx, which is hardcoded with `as`:

```
export default Dropdown as DropdownComponent;
```

in order to workaround https://stackoverflow.com/questions/58469229/react-with-typescript-generics-while-using-react-forwardref.